tests/warnings/semantic/UNDEFINED_DECORATOR.py

```diff
@@ -0,0 +1,5 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring/
+
+@toto
```

Comment:
You can change this line to:
```
@toto # pylint: disable=undefined-variable
```
to disable `undefined-variable` for this line. If there was an `undefined-decorator` pylint flag then I would say disable it for the whole file, but in this case I'd rather just disable it for this line